### PR TITLE
kysely join and many-to-many section

### PR DESCRIPTION
**Pull request by kostasb**
Examples for managing JOINs and many-to-many with SQL and Kysely.

### CODE REVIEW

**Review by SferaDev — COMMENTED** [review]
I would show the Posts Author User example without a junction table first, and then explain junction tables. For a lot of scenarios you might not need them.

**Comment by kostasb** [comment]
Thanks for the feedback! I extended it with an example without junction table plus output samples from all methods kysely, sdk, sql.

**Comment by kostasb** [comment]
All comments merged @joan-ing, let me know if there's anything else!

File: 040-SDK/101-get.mdx [hunk 1]
```diff
@@ -407,6 +407,205 @@ user = xata.records().get("Posts", "myid", columns=["author.*"])
...
+```
+
+```jsonc
+SELECT name,title FROM "posts"
+INNER JOIN "authors" ON "posts".author = "authors".id
+```
+
+</TabbedCode>
+
+Example output:
```
[the remark this answers is not in the document]
Comment (kostasb) [reply 040-SDK/101-get.mdx]: Done! Check it out here under "Example output:": https://xata-docs-pr-179.vercel.app/docs/sdk/get#multiple-relationship-records-per-table

**Comment by kostasb** [comment]
> This looks great @kostasb ! Sorry it took so long to get the review in.

Thanks a lot for the detailed review as always!

**Pull request merged**